lib/agora/active_stream.dart

```diff
@@ -1,18 +1,29 @@
 import 'dart:async';
-import 'dart:math';
+import 'dart:math' as math;
 
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
+import 'package:thc/agora/livestream_button.dart';
 import 'package:thc/home/home_screen.dart';
-import 'package:thc/home/stream/create_livestream.dart';
+import 'package:thc/home/surveys/survey_questions.dart';
+import 'package:thc/home/surveys/take_survey/survey.dart';
 import 'package:thc/utils/app_config.dart';
 import 'package:thc/utils/navigator.dart';
-import 'package:thc/utils/theme.dart';
 import 'package:thc/utils/widgets/state_async.dart';
 
 class ActiveStream extends StatefulWidget {
   const ActiveStream({super.key});
 
+  static const _duration = Durations.extralong1;
+  static PageRouteBuilder get route => PageRouteBuilder(
+        transitionDuration: _duration,
+        reverseTransitionDuration: _duration,
+        pageBuilder: (_, animation, __) => ChangeNotifierProvider(
+          create: (_) => StreamOverlayFadeIn(animation),
+          child: const ActiveStream(),
+        ),
+      );
```

Comment:
We're now using the same widget (fun animations included) for creating and watching livestreams. We're able to tell whether the user is creating or watching by calling `NavBarSelection.of(context).streaming`.